app/api-fetch/song-by-id.ts

```
@@ -0,0 +1,59 @@
+                headers: {
+                    'Cache-Control': 'max-age=300000, stale-while-revalidate',
+                }
+            });
+            localStorage.setItem("song-" + id, JSON.stringify(res.data));
+            localStorage.setItem("songTime-" + id, Date.now().toString());
+            const data = SongSchema.parse(res.data);
+            return data["data"] as SongDetails;
+        }
+    } catch {
+        localStorage.removeItem("song-" + id);
+        localStorage.removeItem("songTime-" + id);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)